doc/release-notes-1.9.rst

```diff
@@ -2,6 +2,42 @@
 
 .. _zephyr_1.9:
 
+Zephyr Kernel 1.9.2
+###################
+
+This is a maintenance release with fixes.
```

Comment:
Similarly (as below):

     This is a maintenance release with fixes summarized below.
     For more details, see the `GitHub 1.9.2 release page`_.

     .. _GitHub 1.9.2 release page:
     https://github.com/zephyrproject-rtos/zephyr/releases/tag/zephyr-v1.9.2